scripts/build-initrd.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+source scripts/common.sh
+
+readonly BUSYBOX_VER=1.34.0
+readonly BUSYBOX_DL="https://busybox.net/downloads/busybox-${BUSYBOX_VER}.tar.bz2"
+readonly BUSYBOX_ARCHIVE=busybox.tar.bz2
+
+function download_busybox()
+{
+    echo "Downloading Busybox version ${BUSYBOX_VER}..."
+    curl -L -o ${OUT}/${BUSYBOX_ARCHIVE} ${BUSYBOX_DL} || exit 1
+    echo "Extracting..."
+    tar -xjf ${OUT}/${BUSYBOX_ARCHIVE} -C ${OUT} || exit 1
+}
+
+function build_busybox()
+{
+   echo "Building Busybox..."
+   cp -f scripts/busybox.config ${OUT}/busybox-${BUSYBOX_VER}/.config
+   make -j `nproc` -C ${OUT}/busybox-${BUSYBOX_VER} all install
```

Comment:
Append `|| exit 1` at the end of this command, so that it can raise errors earlier.